Svelte 5 compatibility

Hello, and thank you for developing this !
I'm migrating my projects to Svelte 5, but svelte-undo doesn't play nice with it :)
Is this just a dependency declaration or is there an actual problem using it with Svelte 5 ?

Sorry I should have poster this in the first post..

```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: goldenboard@0.0.1
npm error Found: svelte@5.2.3
npm error node_modules/svelte
npm error   dev svelte@"^5.0.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peer svelte@"^3.55.1 || ^4.0.0" from @gira-de/svelte-undo@1.2.2
npm error node_modules/@gira-de/svelte-undo
npm error   dev @gira-de/svelte-undo@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\bkupe\AppData\Local\npm-cache\_logs\2024-11-18T23_07_32_490Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\bkupe\AppData\Local\npm-cache\_logs\2024-11-18T23_07_32_490Z-debug-0.log
```

--- a/.changeset/hip-buckets-explain.md
+++ b/.changeset/hip-buckets-explain.md
@@ -1,0 +1,5 @@
+---
+"@gira-de/svelte-undo": minor
+---
+
+add support for Svelte 5

--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
     "immer": "^10.1.1"
   },
   "peerDependencies": {
-    "svelte": "^3.55.1 || ^4.0.0"
+    "svelte": "^3.55.1 || ^4.0.0 || ^5.0.0"
   },
   "repository": {
     "type": "git",

